pallets/phala/src/compute/computation.rs

```diff
@@ -586,8 +586,9 @@ pub mod pallet {
 				block_number > LastBugdetUpdateBlock::<T>::get(),
 				Error::<T>::BudgetUpdateBlockInvalid,
 			);
+			// Double the maxbudgetlimit to compensate budget miss-caculated during the last nonce.
 			ensure!(
-				budget <= MaxBudgetLimit::<T>::get(),
+				budget <= MaxBudgetLimit::<T>::get() * 2,
```

Comment:
Do you think it's better to adjust MaxBudgetLimit directly instead of adding a magic number here?

Reply:
I've thought about it, but max budget limit should be the original one, the double limit is just an estimated limit for block production speed is lower than we expect.

Reply:
So i also added some comment to introduce the magic number.